app/graphql/types/objects/consultation_response/base.rb

```diff
@@ -33,6 +34,10 @@ def user
 					return object.user if object.user == context[:current_user]
 					return nil
 				end
+
+				def round_number
```

Comment:
@BALAJI-BALA - how about making this a method in the model?

Reply:
@mikevic cool bro fixing this